src/Atmos/Model/source.jl

```diff
@@ -33,6 +33,24 @@ function atmos_source!(::Coriolis, atmos::AtmosModel, source::Vars, state::Vars,
   source.ρu -= SVector(0, 0, 2*Omega) × state.ρu
 end
 
+struct Subsidence{FT} <: Source
+  D::FT
+end
+# TODO: Add transform::Grad, or whatever is needed.
+# function atmos_source!(subsidence::Subsidence, atmos::AtmosModel, source::Vars, state::Vars, aux::Vars, transform::Grad, t::Real)
+function atmos_source!(subsidence::Subsidence, atmos::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
+  ρ = state.ρ
+  z = altitude(atmos.orientation, aux)
+  w_sub = subsidence_velocity(subsidence, z)
+  k̂ = vertical_unit_vector(atmos.orientation, aux)
+
+  source.ρe += ρ*w_sub .* k̂ .* diffusive.∇h_tot
+  source.moisture.ρq_tot += ρ*w_sub .* k̂ .* diffusive.moisture.∇q_tot
+end
+
+subsidence_velocity(subsidence::Subsidence{FT}, z::FT) where {FT} = subsidence.D*z
```

Comment:
I think this should have a minus sign @tapios @charleskawczynski  ? can you confirm

Reply:
I think we have `subsidence_velocity` defined as positive downward, in which case the sign is correct. This definition makes sense because subsidence implies it is down. However, it may be confusing. I'd be fine with changing the sign of the subsidence velocity so it is positive upward, and then changing the sign of the source term correspondingly.

Reply:
Based on the design docs, you're correct. But I also added this term (which should be negative), so the source turns out to be correct. I'll fix this.